lib/IRGen/GenMeta.cpp

```diff
@@ -729,6 +729,20 @@ namespace {
               B.getAddrOfCurrentPosition(IGM.ProtocolRequirementStructTy));
         }
 
+        if (entry.isBase()) {
+          // Define the associated conformance descriptor to point to the
```

Comment:
"Define a base conformance descriptor, which is just an associated conformance descriptor which etc."